class/class.2.1/hello.cpp

```diff
@@ -0,0 +1,83 @@
+#include <iostream>
+#include <fstream>
+#include <unordered_map>
+#include <string>
+
+const std::string FILENAME = "user.txt";
+
+class UserStats {
+public:
+    void load() {
+        std::ifstream infile(FILENAME);
+        if (!infile) {
+            std::cerr << "Failed to open file for reading.\n";
+            return;
+        }
+
+        std::string name;
+        int count;
+        while (infile >> name >> count) {
+            stats[name] = count;
+        }
+        std::cout << "Loaded stats from file.\n";
```

Comment:
std::println(""Loaded stats from file.");
also it is better to generate different messages on the level of the code which calls the class - it can be done for example with std::expected, std::optional or just bool as a return type